src/object/camera.cpp

```diff
@@ -30,49 +30,49 @@
 #include "video/video_system.hpp"
 #include "video/viewport.hpp"
 
-/* this is the fractional distance toward the peek
+/* This is the fractional distance toward the peek
    position to move each frame; lower is slower,
-   0 is never get there, 1 is instant */
+   0 is never get there, 1 is instant. */
 static const float PEEK_ARRIVE_RATIO = 0.03f;
 
 /**
  * For the multiplayer camera, the camera will ensure all players are visible.
  * These variables allow establishing a minimum zone around them that will also
  * be always visible.
  */
-static const float HORIZONTAL_MARGIN = 196.f; // 6 tiles
-static const float VERTICAL_MARGIN = 196.f; // 6 tiles
-static const float PEEK_ADD_HORIZONTAL_MARGIN = 320.f; // 10 tiles
-static const float PEEK_ADD_VERTICAL_MARGIN = 320.f; // 10 tiles
+static const float HORIZONTAL_MARGIN = 196.f; // 6 tiles.
+static const float VERTICAL_MARGIN = 196.f; // 6 tiles.
+static const float PEEK_ADD_HORIZONTAL_MARGIN = 320.f; // 10 tiles.
+static const float PEEK_ADD_VERTICAL_MARGIN = 320.f; // 10 tiles.
 
-/* 0 = no movement, 1 = no smooth adaptation */
+/* 0 = no movement, 1 = no smooth adaptation. */
 static const float MULTIPLAYER_CAM_WEIGHT = 0.1f;
 
 class CameraConfig final
 {
 public:
-  // 0 = No, 1 = Fix, 2 = Mario/Yoshi, 3 = Kirby, 4 = Super Metroid-like
+  // 0 = No, 1 = Fix, 2 = Mario/Yoshi, 3 = Kirby, 4 = Super Metroid-like.
   int xmode;
-  // as above
+  // As above.
   int ymode;
   float kirby_rectsize_x;
   float kirby_rectsize_y;
-  // where to fix the player (used for Yoshi and Fix camera)
+  // Where to fix the player (used for Yoshi and Fix camera).
```

Comment:
References to other games in comments should probably be avoided. Not clear for everybody.